src/blob/core.rs

```diff
@@ -322,8 +322,11 @@ where
     }
 
     #[inline]
-    pub(crate) async fn read_all_entries(&self, key: &K) -> Result<ReadResult<Vec<Entry>>> {
```

Comment:
I think it would be good to have `read_all_entries` on the Blob level too (in addition to `read_all_entries_with_deletion_marker`)